components/list/demo/demo-list-nested.js

```diff
@@ -197,6 +197,7 @@ class ListDemoNested extends LitElement {
 		return html`
 			<d2l-list-item
 				action-href="${this.includeActionHref ? 'http://www.d2l.com' : ''}"
+				color="#ffaa47"
```

Comment:
Thoughts on if the # should be included by the consumer or if we should be adding that?

Reply:
My first thought is that they should be required to provide the `#` and we shouldn't assume anything. Although we might decide to limit this to hex colors, technically we could allow `rgb`, `rgba`, named colors, CSS properties, etc. and we probably shouldn't assume they'll never be accepted.